utility/script_config_dict.py

```diff
@@ -0,0 +1,430 @@
+            'filenames have a <name>@<revision>.yang format. If there are new RFC files, produce an automated message '
+            'that will be sent to the Cisco Webex Teams and admin emails notifying that these need to be added to the '
+            'YangModels/yang GitHub repository manually. This script runs as a daily cronjob. '
+        ),
+        'args': [
+            {
+                'flag': '--config-path',
+                'help': 'Set path to config file',
+                'type': str,
+                'default': os.environ['YANGCATALOG_CONFIG_PATH'],
+            },
+            {
+                'flag': '--send-message',
+                'help': 'Whether to send a notification',
+                'action': 'store_true',
+                'default': False,
+            },
+        ],
+    },
+    'draftPullLocal': {
+        'help': (
+            'Run populate script on all ietf RFC and DRAFT files to parse all ietf modules and populate the '
+            'metadata to yangcatalog if there are any new. This runs as a daily cronjob'
+        ),
+        'args': [
+            {
+                'flag': '--config-path',
+                'help': 'Set path to config file',
+                'type': str,
+                'default': os.environ['YANGCATALOG_CONFIG_PATH'],
+            },
+        ],
+    },
+    'ianaPull': {
+        'help': 'Pull the latest IANA-maintained files and add them to the Github if there are any new.',
+        'args': [
+            {
+                'flag': '--config-path',
+                'help': 'Set path to config file',
+                'type': str,
+                'default': os.environ['YANGCATALOG_CONFIG_PATH'],
+            },
+        ],
+    },
+    'openconfigPullLocal': {
+        'help': (
+            'Run populate script on all openconfig files to parse all modules and populate the'
+            ' metadata to yangcatalog if there are any new. This runs as a daily cronjob'
+        ),
+        'args': [
+            {
+                'flag': '--config-path',
+                'help': 'Set path to config file',
+                'type': str,
+                'default': os.environ['YANGCATALOG_CONFIG_PATH'],
+            },
+        ],
+    },
+    'integrity': {
+        'help': '',
+        'args': [
+            {
+                'flag': '--sdo',
+                'help': 'If we are processing sdo or vendor yang modules',
+                'action': 'store_true',
+                'default': False,
+            },
+            {
+                'flag': '--dir',
+                'help': 'Set directory where to look for hello message xml files',
+                'type': str,
+                'default': '/var/yang/nonietf/yangmodels/yang/standard/ietf/RFC',
+            },
+            {'flag': '--output', 'help': 'Output json file', 'type': str, 'default': 'integrity.json'},
+        ],
+        'yang_models': config.get('Directory-Section', 'yang-models-dir'),
```

Comment:
I think this isn't meant to be here.

Reply:
That info was inside previous ScriptConfig for this file, I checked it is not used in the integrity.py, and it works without it, so yeah, I guess I can remove it